src/ast/mod.rs

```diff
@@ -1541,7 +1541,10 @@ pub enum Statement {
     /// Note: This is a MySQL-specific statement.
     Use { db_name: Ident },
     /// `{ BEGIN [ TRANSACTION | WORK ] | START TRANSACTION } ...`
-    StartTransaction { modes: Vec<TransactionMode> },
+    StartTransaction {
+        modes: Vec<TransactionMode>,
+        syntax_begin: bool,
```

Comment:
Could we perhaps add a comment explaining what `syntax_begin` means (so that people don't have to look at the source code?)

Maybe 
```
    /// `START  [ TRANSACTION | WORK ] | START TRANSACTION } ...`
    /// If `syntax_begin` is false.
    ///
    /// `BEGIN  [ TRANSACTION | WORK ] | START TRANSACTION } ...`
    /// If `syntax_begin` is true
````

?

Reply:
Also, calling this field `begin` might be more consistent with the rest of this crate but `syntax_begin` is fine as well